examples_tests/44.LevelCurveExtraction/main.cpp

```diff
@@ -30,40 +30,118 @@ void main()
 
 const char* geometryShaderCode = R"===(
 #version 440 core
+
+#define FLT_MAX 3.402823466e+38
 struct DrawIndirectArrays_t
 {
-	uint  count;
-	uint  instanceCount;
-	uint  first;
-	uint  baseInstance;
+    uint  count;
+    uint  instanceCount;
+    uint  first;
+    uint  baseInstance;
 };
```

Comment:
there are engine built-ins for that now, if you merge the latest `shader_pipeline`